stream-chat-android-ui-guides/src/main/java/io/getstream/chat/android/guides/login/LoginActivity.kt

```diff
@@ -0,0 +1,198 @@
+                painter = painterResource(id = R.drawable.ic_stream),
+                contentDescription = null,
+                tint = ChatTheme.colors.primaryAccent,
+            )
+
+            Spacer(modifier = Modifier.height(28.dp))
+
+            LazyColumn(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .weight(1f),
+            ) {
+                items(items = LoginUsers.createUsers()) { loginUser ->
+                    UserItem(loginUser = loginUser)
+
+                    DividerItem()
+                }
+            }
+        }
+    }
+
+    /**
+     * Represents a user whose credentials will be used for login.
+     *
+     * @param loginUser The user credentials to login with.
+     */
+    @Composable
+    fun UserItem(loginUser: LoginUser) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(64.dp)
+                .clickable(
+                    onClick = { login(loginUser) },
+                    indication = rememberRipple(),
+                    interactionSource = remember { MutableInteractionSource() }
+                )
+                .padding(horizontal = 16.dp),
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            UserAvatar(
+                modifier = Modifier.size(40.dp),
+                user = loginUser.user,
+            )
+
+            Column(
+                modifier = Modifier
+                    .wrapContentHeight()
+                    .weight(1f)
+                    .padding(horizontal = 16.dp),
+            ) {
+                Text(
+                    text = loginUser.user.name,
+                    fontSize = 14.sp,
+                    fontWeight = FontWeight.Bold,
+                    color = ChatTheme.colors.textHighEmphasis
+                )
+
+                Text(
+                    text = stringResource(id = R.string.login_user_subtitle),
+                    fontSize = 12.sp,
+                    color = ChatTheme.colors.textLowEmphasis
+                )
+            }
+
+            Icon(
+                modifier = Modifier.wrapContentSize(),
+                painter = painterResource(id = R.drawable.ic_arrow),
+                contentDescription = null,
+                tint = ChatTheme.colors.primaryAccent
+            )
+        }
+    }
+
+    /**
+     * Represents a separator between user items.
+     */
+    @Composable
+    private fun DividerItem() {
+        Spacer(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(0.5.dp)
+                .background(color = ChatTheme.colors.borders)
+        )
+    }
+
+    /**
+     * Initializes the SDK with the given user and navigates to the catalog screen.
+     *
+     * @param loginUser The user credentials to login with.
+     */
+    private fun login(loginUser: LoginUser) {
+        ChatClient.instance().connectUser(
```

Comment:
The idea is that guides and samples should share as much logic as possible, so SDK initilization and logic is done only once. So, if we need a new guide we don't need to care about that.